catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/VariableBrickTest.java

```diff
@@ -102,4 +102,18 @@ public void testNewVariableCanceling() {
 		checkIfBrickAtPositionShowsString(setBrickPosition, R.string.brick_set_variable);
 		checkIfBrickAtPositionShowsString(changeBrickPosition, R.string.brick_change_variable);
 	}
+
+	@Test
+	public void testAfterDeleteBrickVariableStillVisible() {
+		final String variableName = "testVariable";
+		onView(withId(R.id.set_variable_spinner)).perform(click());
```

Comment:
please access spinners on bricks by also stating the brick. (especially in test cases where theres more than one brick in the script)